compiler/noirc_evaluator/src/ssa/opt/normalize_value_ids.rs

```diff
@@ -25,7 +26,8 @@ impl Ssa {
         let mut context = Context::default();
         context.populate_functions(&self.functions);
         for function in self.functions.values_mut() {
-            context.normalize_ids(function, &self.globals);
+            let globals = (*function.dfg.globals).clone();
```

Comment:
Is this `clone` before passing a reference avoidable?

Reply:
It is not as `GlobalsGraph` cannot be moved out of `Arc` as it does not implement `Copy`. I have moved it out of the loop though.